star-wars-browser/src/store/people/saga.js

```diff
@@ -0,0 +1,24 @@
+import { call, put, takeEvery } from 'redux-saga/effects';
+import { fetchPeople, setPeople } from './actions';
+import { SWAPI_ADDRESS } from '../../config';
+
+export function* fetchPeopleSaga() {
+  const peopleArray = [];
+  const response = yield call(fetch, `${SWAPI_ADDRESS}`);
+  let data = yield response.json();
```

Comment:
dobrze jest rozdzielać pustymi liniami blok deklaracji od reszty (tutaj dodaj postą linię po let)

dodatkowo jak masz jakiś blok, to też staraj się przed nim i po nim dać pustą linię (np. jak jest if albo while - jest tutaj poniżej)

Reply:
poprawione, poszukałem również w innych miejscach tego błędu